Cargo.toml

```
@@ -73,7 +73,7 @@ name = "api_trait_implementation"
 required-features = ["telegram-trait"]
 
 [dependencies]
-bon = "2.2.0"
+bon = "3.1.0"
```

see https://github.com/ayrat555/frankenstein/pull/231#discussion_r1854200522 for original comment

```suggestion
bon = "3.0.0"
```